package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "city-in-the-south",
+  "version": "1.0.0",
+  "description": "## PROBLEM:",
+  "main": "index.js",
+  "scripts": {
+    "server": "fuser -k 5000/tcp;clear; nodemon server/index.js",
+    "client": "npm start --prefix client",
```

Comment:
```suggestion
    "start:client": "npm start --prefix client",
```

Reply:
I think it is ok, no need for extra word!